pwa/src/data/mimeTypes.ts

```diff
@@ -0,0 +1,84 @@
+  {name: 'application/ld+json', value: 'application/ld+json'},
+  {name: 'audio/midi audio/x-midi', value: 'audio/midi audio/x-midi'},
+  {name: 'text/javascript', value: 'text/javascript'},
+  {name: 'audio/mpeg', value: 'audio/mpeg'},
+  {name: 'video/mp4', value: 'video/mp4'},
+  {name: 'video/mpeg', value: 'video/mpeg'},
+  {name: 'application/vnd.apple.installer+xml', value: 'application/vnd.apple.installer+xml'},
+  {name: 'application/vnd.oasis.opendocument.presentation', value: 'application/vnd.oasis.opendocument.presentation'},
+  {name: 'application/vnd.oasis.opendocument.spreadsheet', value: 'application/vnd.oasis.opendocument.spreadsheet'},
+  {name: 'application/vnd.oasis.opendocument.text', value: 'application/vnd.oasis.opendocument.text'},
+  {name: 'audio/ogg', value: 'audio/ogg'},
+  {name: 'video/ogg', value: 'video/ogg'},
+  {name: 'application/ogg', value: 'application/ogg'},
+  {name: 'audio/opus', value: 'audio/opus'},
+  {name: 'font/otf', value: 'font/otf'},
+  {name: 'image/png', value: 'image/png'},
+  {name: 'application/pdf', value: 'application/pdf'},
+  {name: 'application/x-httpd-php', value: 'application/x-httpd-php'},
+  {name: 'application/vnd.ms-powerpoint', value: 'application/vnd.ms-powerpoint'},
+  {
+    name: 'application/vnd.openxmlformats-officedocument.presentationml.presentation',
+    value: 'application/vnd.openxmlformats-officedocument.presentationml.presentation'
+  },
+  {name: 'application/vnd.rar', value: 'application/vnd.rar'},
+  {name: 'application/rtf', value: 'application/rtf'},
+  {name: 'application/x-sh', value: 'application/x-sh'},
+  {name: 'image/svg+xml', value: 'image/svg+xml'},
+  {name: 'application/x-shockwave-flash', value: 'application/x-shockwave-flash'},
+  {name: 'application/x-tar', value: 'application/x-tar'},
+  {name: 'image/tiff', value: 'image/tiff'},
+  {name: 'video/mp2t', value: 'video/mp2t'},
+  {name: 'font/ttf', value: 'font/ttf'},
+  {name: 'text/plain', value: 'text/plain'},
+  {name: 'application/vnd.visio', value: 'application/vnd.visio'},
+  {name: 'audio/wav', value: 'audio/wav'},
+  {name: 'audio/webm', value: 'audio/webm'},
+  {name: 'video/webm', value: 'video/webm'},
+  {name: 'image/webp', value: 'image/webp'},
+  {name: 'font/woff', value: 'font/woff'},
+  {name: 'font/woff2', value: 'font/woff2'},
+  {name: 'application/xhtml+xml', value: 'application/xhtml+xml'},
+  {name: 'application/vnd.ms-excel', value: 'application/vnd.ms-excel'},
+  {name: 'application/vnd', value: 'application/vnd'},
+  {name: 'application/xml', value: 'application/xml'},
+  {name: 'application/vnd.mozilla.xul+xml', value: 'application/vnd.mozilla.xul+xml'},
+  {name: 'application/zip', value: 'application/zip'},
+  {name: 'video/3gpp; audio/3gpp', value: 'video/3gpp; audio/3gpp'},
+  {name: 'video/3gpp2; audio/3gpp2', value: 'video/3gpp2; audio/3gpp2'},
+  {name: 'application/x-7z-compressed', value: 'application/x-7z-compressed'},
+]
```

Comment:
I think we should rename all the `name` props of this object. We discussed earlier that we would do it the way you've built it now. However, I now realise that it's really hard to filter as a user (it's not alphabetically ordered right now).

However, the [link we used](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types) is alphabetically ordered on the .extension. 

Therefore, I think we should rename all the `name` props to the `.extension` value. Like so:


```
{ name: ".aac", value: "audio/aac" },
{ name: ".abw", value: "application/x-abiword" },
```